odoo/addons/base/models/ir_model.py

```diff
@@ -1682,7 +1682,7 @@ class IrModelAccess(models.Model):
     _description = 'Model Access'
     _order = 'model_id,group_id,name,id'
 
-    name = fields.Char(required=True, index=True)
+    name = fields.Char(compute='_compute_name', store=True, readonly=False)
```

Comment:
I would remove it altogether, and add a useful search field instead on "Groups Implied by".
For instance, if you search for `account.group_account_manager`, you will also see everything related to `account.group_account_invoice` and `base.group_private_addresses`, but also `base.group_user` (through `account.group_account_invoice`)